charts/konk/templates/kind-pod.yaml

```
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ include "konk.fullname" . }}-kind-pod
```

There's got to be a better name for this... maybe init?
```suggestion
  name: {{ include "konk.fullname" . }}-init
```

---

Changed as per suggestion.